stdlib/runtime/malloc.gr

```diff
@@ -255,7 +258,13 @@ export let malloc = (nb: WasmI32) => {
   let mut ret = -1n
 
   // Search the freelist for any blocks large enough.
-  for (let mut p = getNext(prevp); ; {prevp = p; p = getNext(p)}) {
+  for (
+    let mut p = getNext(prevp);;
+    {
+      prevp = p
+      p = getNext(p)
+    }
```

Comment:
This now breaks weird, with the double colon on the line above. I completely missed it visually and needed to double-take.

Reply:
This is caused by the block being forced open now.   How would you write this manually with the new open block?

Reply:
fixed.